Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Solver.cs

```diff
@@ -18,24 +19,23 @@ public abstract class Solver : MonoBehaviour
         private bool updateLinkedTransform = false;
 
         [SerializeField]
-        [Tooltip("Position lerp multiplier")]
+        [Tooltip("If 0, the position will update immediately.  Otherwise, the greater this attribute the slower the position updates")]
         private float moveLerpTime = 0.1f;
 
         [SerializeField]
-        [Tooltip("Rotation lerp multiplier")]
+        [Tooltip("If 0, the rotation will update immediately.  Otherwise, the greater this attribute the slower the rotation updates")]
         private float rotateLerpTime = 0.1f;
 
         [SerializeField]
-        [Tooltip("Scale lerp multiplier")]
+        [Tooltip("If 0, the scale will update immediately.  Otherwise, the greater this attribute the slower the scale updates")]
         private float scaleLerpTime = 0;
 
         [SerializeField]
         [Tooltip("If true, the Solver will respect the object's original scale values")]
         private bool maintainScale = true;
-
         [SerializeField]
-        [Tooltip("Working output is smoothed if true. Otherwise, snapped")]
-        private bool smoothing = true;
+        [Tooltip("If true, updates are smoothed to the target. Otherwise, they are snapped to the target")]
+        public bool smoothing = true;
```

Comment:
Per the [MRTK Coding Guidelines](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html), it would be great if we could keep this field private, and add a public getter and sett

Reply:
Line 38 contains the private field, so this looks like a duplicately named field and should probably be removed.